tests/Event/EventTest.php

```diff
@@ -535,6 +535,267 @@ function (Event $event) use ($bookingInfo) {
+                                new Currency('EUR')
+                            )
+                        ),
+                    ])
+                )
+            )
+            ->then([
+                new PriceInfoUpdated(
+                    'd2b41f1d-598c-46af-a3a5-10e373faa6fe',
+                    (new PriceInfo(new BasePrice(new Money(90, new Currency('EUR')))))
+                        ->withUiTPASTariffs([
+                            new \CultuurNet\UDB3\PriceInfo\Tariff(
+                                new MultilingualString(
+                                    new LegacyLanguage('nl'),
+                                    new StringLiteral('Tariff 1')
+                                ),
+                                new Money(
+                                    199,
+                                    new Currency('EUR')
+                                )
+                            ),
+                        ])
+                ),
+            ]);
+    }
+
+    /**
+     * @test
+     */
+    public function it_ignores_an_update_with_equal_prices_on_uitpas_prices(): void
+    {
+        $this->scenario
+            ->given([
+                $this->getCreationEvent(),
+                new PriceInfoUpdated(
+                    'd2b41f1d-598c-46af-a3a5-10e373faa6fe',
+                    new PriceInfo(
+                        new BasePrice(
+                            new Money(
+                                100,
+                                new Currency('EUR')
+                            )
+                        )
+                    )
+                ),
+                new PriceInfoUpdated(
+                    'd2b41f1d-598c-46af-a3a5-10e373faa6fe',
+                    (new PriceInfo(new BasePrice(new Money(100, new Currency('EUR')))))
+                        ->withUiTPASTariffs([
+                            new \CultuurNet\UDB3\PriceInfo\Tariff(
+                                new MultilingualString(
+                                    new LegacyLanguage('nl'),
+                                    new StringLiteral('Tariff 1')
+                                ),
+                                new Money(
+                                    199,
+                                    new Currency('EUR')
+                                )
+                            ),
+                        ])
+                ),
+            ])
+            ->when(
+                fn (Event $event) => $event->updatePriceInfo(
+                    new PriceInfo(
+                        new BasePrice(
+                            new Money(
+                                100,
+                                new Currency('EUR')
+                            )
+                        )
+                    )
+                )
+            )
+            ->then([]);
+    }
+
+    /**
+     * @test
+     */
+    public function it_ignores_an_update_with_equal_prices_with_uitpas_on_uitpas_prices(): void
```

Comment:
I'm not sure what the `_with_uitpas_on_uitpas_prices` means. Copy/paste mistake from the previous test?